### ISSUE TITLE
Null Claim - Veteran File Number and SSN do not match

### ISSUE THREAD

**Issue by tblackwe** [department-of-veterans-affairs/va.gov-team#64138]
If a Veteran's file number and SSN do not match, we upload a PDF copy of their claim but then throw an exception. This is a silent failure for the Veteran and results in a Null Claim

**Comment by tblackwe** [comment]
@tfink419 could you help me identify where in `vets-api` we get the file number for the Vet?

**Comment by data-doge** [comment]
Here's the context I have on File # / SSN mismatches:
- @tblackwe, I sent you a Teams message with a bunch of screenshots of Teams conversations I had on this topic.
- See https://github.com/department-of-veterans-affairs/va.gov-team/issues/56995 and its comments
- See https://github.com/department-of-veterans-affairs/va.gov-team/issues/57798 and its comments

**TL;DR:** See https://github.com/department-of-veterans-affairs/va.gov-team/issues/56995#issuecomment-1532129932. File # / SSN mismatches are a big, complicated, systemic issue. IMO, the best we can do is:
1. Report them to MPI to resolve. See https://github.com/department-of-veterans-affairs/va.gov-team/issues/57798. 
2. Fail loudly when we discover a File # / SSN mismatch before jobs are enqueued, and render an appropriate error on the front-end, so that the veteran can contact VA for help remedying their user data.
3. Confirm that error notification emails are sent to veterans when we discover a File # / SSN mismatch while one of our jobs are running.

**Comment by tblackwe** [comment]
your TL;DR matches with my own thoughts. You've done a significant amount of due diligence (thank you so much!) that explains why they can be different, but it also showed that this problem needs to be resolved at a higher level.

I agree with your 3 step plan. I'm researching how early in the flow we can detect SSN - File Number mismatch so that we can alert early and not have the veteran create a claim that will ultimately error. I am also going to discuss with Design/Research how we can prevent mismatch from the Veteran manually inputing spouse information.

**Comment by tblackwe** [comment]
If I reuse the same error as `Invalid File Number` the error we get is:
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbEUrIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--55f3416bcbb8e5169819bfaa45e3e51596fead32/image.png)


The Veteran would see this as soon as they navigated to the add/remove URL. 

@djones2113 @juliepedtke Can you have a think on this error message? We can easily use different copy if required.

**Comment by tblackwe** [comment]
@djones2113 sounds good. let me know when it is finalized and I will open the Pull Request.

**Comment by djones2113** [comment]
@tblackwe after talking with Laura and Julie P this morning, here are our thoughts:

Since this is a minor change, we think the next logical step would be sharing this change with Julie S for feedback. However, she won't be available to review this until Tuesday at the earliest since she is out this week. 

If this issue is very high priority and shouldn't wait until Julie S is back, I think we can proceed with the changes I recommended above and then just plan to check in with her when she returns. Thoughts?

**Comment by tblackwe** [comment]
It can wait for Julie S. She should sign off on the approach and verbiage.

**Comment by tblackwe** [comment]
@djones2113 that error message is in our scope. I've fixed it in my drafted change.

**Comment by jstrothman** [comment]
@tblackwe @djones2113 I suspect you already know this from today's discussion and issue #64137 -- but the proposed error msg  won't always be accurate. Pasting in notes from today's discussion:

- There are four or five variations of what a file number might be, and only one variation would match SSN. Depends on age of Veteran and when they served. This is less and less relevant, it was mostly before Vietnam era that the numbers deviated.
- Currently, downstream system (BGS) enforces that these numbers be the same - we should start a request to get this changed
- (Sensitive file won’t impact their file number)
- How often is this happening? Two cases: if the Veteran has different file # and SSN, does not happen frequently on 686. When we ask for spouse’s SSN on 686 we see more issues.
- File number is used for searching and identifying the Veteran
  - MPI includes file number, SSN, EDIPI, PID… tries to associate all of these to the Veteran. For older Veterans, the numbers may be different. There’s an effort to clean up the MPI, millions of records aren’t associated with the correct file number or SSN, often run into problems with mismatches.
  - “If applicable” on the Pension form refers to if the SSN and file number are the same, there's no need to re-enter it in this case. But in cases where it’s different from SSN, it would be helpful to have both

**Comment by tblackwe** [comment]
as discussed on call with @emilytheis and @jstrothman, Julie S will draft a new version of the error message to use as a stop gap.
@steele-lm will create ticket to begin conversations about changing backend requirements

**Comment by jstrothman** [comment]
If we detect that a Veteran has different File Number and SSN we'll block submission and instruct the Veteran to submit paper version, let's use this text for the error message:

Error heading:
> Unfortunately, you'll need to submit a paper version of this 686c form

Error content:
> The online form is unable to process submissions where the Veteran's Social Security Number doesn't match the Veteran's File Number. We're working to address this issue, but we don't want to delay your submission. Please visit [About VA Form 21-686c](https://www.va.gov/find-forms/about-form-21-686c/) to download a pdf you can print.

**Comment by jstrothman** [comment]
@tblackwe @steele-lm Forgot to mention you in comment above w/error message wording

**Comment by steele-lm** [comment]
Follow on ticket to uncover root cause an approach to remediating: https://github.com/department-of-veterans-affairs/va.gov-team/issues/65054

**Comment by tblackwe** [comment]
When we get Backup Submissions to Central Mail processing, SSN mismatch claims will start getting processed by CMS. We can leave the forms as-is for now, while requesting change from BIS to resolve backend validation issue.
